ksp_plugin/part.cpp

```diff
@@ -63,6 +72,10 @@ PartId Part::part_id() const {
   return part_id_;
 }
 
+bool Part::loaded() const {
+  return loaded_;
+}
+
 void Part::set_mass(Mass const& mass) {
```

Comment:
`loaded_` should be set once we get actual data from `SetApparent...`.
Perhaps for symmetry `InsertOrKeepLoadedPart` should set it.